### PR TITLE
SPM and Cocoapods Support

### PR DESCRIPTION
**Pull request by Sergey70**
contrib

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        .library(name: "protobuf-c", targets: ["protobuf-c"])
+    ],
+    targets: [
+        .target(
+            name: "protobuf-c",
+            path: "protobuf-c",
+            cSettings: [
+                .headerSearchPath("./include/protobuf-c"),
+                .unsafeFlags([
+                    "-Wno-conversion"
```
Comment (GLinnik21) [inline Package.swift:18]: SPM would not allow this flag with release version

**Review by GLinnik21 — CHANGES_REQUESTED** [review]

File: Package.swift (hunk 1 above)
Comment (Sergey70) [reply Package.swift:18]: Any suggestion?

Comment (Sergey70) [reply Package.swift]: Убрал флаг. Проверил в семпле, варнинги, вроде бы, не экспоузятся в сборку самого приложения. Предлагаю позже перепроверить это еще раз.

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// swift-tools-version: 5.6
```
Comment (GLinnik21) [inline Package.swift:1]: Maybe we can downgrade the version of the tools for better compatibility

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+let package = Package(
+    name: "protobuf-c",
+    products: [
+        .library(name: "protobuf-c", targets: ["protobuf-c"])
+    ],
+    targets: [
+        .target(
+            name: "protobuf-c",
+            path: "protobuf-c",
```
Comment (GLinnik21) [inline Package.swift:14]: I would also add `sources:` param here as well, as protobuf-c build toolchain generates some files that could break SPM build.

File: protobuf-c.podspec [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  spec.authors     = { "Dave Benson" => "" }
+  spec.source      = {
+    :git => "https://github.com/protobuf-c/protobuf-c.git",
+    :tag => "v#{spec.version}"
+  }
+  spec.homepage    = "https://github.com/protobuf-c/protobuf-c"
+  spec.license     = { :type => "BSD", :file => "LICENSE" }
+
+  spec.ios.deployment_target     = '9.0'
+  spec.tvos.deployment_target    = '10.0'
```
Comment (GLinnik21) [inline protobuf-c.podspec:14]: Why does iOS support start at 9.0 and tvOS at 10.0?
protobuf-c also supports macOS and watchOS.

**Review by GLinnik21 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**